reactor-netty-core/src/main/java/reactor/netty/tcp/TcpResources.java

```diff
@@ -165,25 +165,54 @@ public boolean daemon() {
 	}
 
 	/**
-	 * Use {@link #disposeLoopsAndConnections()}
+	 * {@link TcpResources} provides the global TCP resources, as such {@link TcpResources} lifecycle is
+	 * longer than regular {@link LoopResources} and {@link ConnectionProvider} implementations lifecycle.
+	 * In order to prevent an unintended disposal of the global TCP resources, {@link #disposeLoopsAndConnections()}
+	 * is exposed and should be used.
+	 * <p>
+	 * This method has {@code NOOP} implementation.
 	 */
 	@Override
 	public void dispose() {
 		//noop on global by default
 	}
 
 	/**
-	 * Use {@link #disposeLoopsAndConnectionsLater()}
+	 * {@link TcpResources} provides the global TCP resources, as such {@link TcpResources} lifecycle is
+	 * longer than regular {@link LoopResources} and {@link ConnectionProvider} implementations lifecycle.
+	 * In order to prevent an unintended disposal of the global TCP resources, {@link #disposeLoopsAndConnectionsLater()}
+	 * is exposed and should be used.
+	 * <p>
+	 * This method has {@code NOOP} implementation.
 	 */
 	@Override
 	public Mono<Void> disposeLater() {
 		//noop on global by default
 		return Mono.empty();
 	}
 
+	/**
+	 * {@link TcpResources} provides the global TCP resources, as such {@link TcpResources} lifecycle is
+	 * longer than regular {@link LoopResources} and {@link ConnectionProvider} implementations lifecycle.
+	 * In order to prevent an unintended disposal of the global TCP resources,
+	 * {@link #disposeLoopsAndConnectionsLater(Duration, Duration)} is exposed and should be used.
+	 * <p>
+	 * This method has {@code NOOP} implementation.
+	 */
+	@Override
+	public Mono<Void> disposeLater(Duration quietPeriod, Duration timeout) {
+		//noop on global by default
+		return Mono.empty();
+	}
+
+	/**
+	 * Dispose all connection pools for the specified remote address.
```

Comment:
maybe specify that unlike eg. `disposeLater`, this method _does_ delegate to the underlying pool and has a global effect (comparable to some extent to `disposeLoopsAndConnectionsLater()`).

Reply:
done

Reply:
suggested a phrasing tweak